var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -217,6 +220,12 @@ def configure_args(self):
         else:
             args.append('--enable-fast=all')
 
+        if '+cuda' in self.spec:
+            args.extend([
+                '--enable-cuda',
+                '--with-cuda={0}'.format(spec['cuda'].prefix)
+            ])
```

Comment:
It seems that `--disable-cuda` is not a valid option hwen builidng mvapich2. CUDA extension is disabled by default in mvapich2. http://mvapich.cse.ohio-state.edu/static/media/mvapich/mvapich2-2.0-userguide.html